app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt

```diff
@@ -90,6 +90,7 @@ class ConversationsFragment : SFragment(), StatusActionListener, Injectable, Res
             useAbsoluteTime = preferences.getBoolean("absoluteTimeView", false),
             showBotOverlay = preferences.getBoolean("showBotOverlay", true),
             useBlurhash = preferences.getBoolean("useBlurhash", true),
+            showDescriptionBadge = preferences.getBoolean("showDescriptionBadge", true),
```

Comment:
Not related to the patch, but is it weird that this very similar block of code to create a StatusDisplayOptions is duplicated so many times? Is it weird that some of the instances use PrefKeys and some use literal strings?

Reply:
I start to tackle this as part of https://github.com/tuskyapp/Tusky/pull/3159/files#diff-795e9e3b0efed324958f1c9b55242bf882405d00c87a7634ce8d57b60f8cc002.